dts/bindings/sensor/nxp,fxos8700.yaml

```diff
@@ -19,6 +19,12 @@ properties:
     compatible:
       constraint: "nxp,fxos8700"
 
+    whoami:
+      type: int
+      description: value of the WHOAMI register
+      generation: define
+      category: required
```

Comment:
can we make it optional, since it was for early devices, and if this isn't set we assume the value is 0xc7

Reply:
Can we still have it as optional, such that it the property is set, it means to set the WHO_AM_I register rather than just reading it.  It seems like the only reason we need this is for some early devices that didn't get WHO_AM_I set by the mfg.

Reply:
I have done it quite differently now